packages/utils/modify-config-helper.ts

```diff
@@ -123,7 +123,12 @@ export default function modifyHelperUtil(
 		);
 		return runTransform(transformConfig, action);
 	}).catch((err) => {
-		console.error(chalk.red("\nUnexpected Error, please file an issue to https://github.com/webpack/webpack-cli\n"));
+		console.error(
+			chalk.red(
+				// tslint:disable-next-line:max-line-length
+				"\nUnexpected Error, please file an issue to https://github.com/webpack/webpack-cli/issues/new?template=Bug_report.md \n",
```

Comment:
Use a template literal instead.
```suggestion
				`
				Unexpected Error
				please file an issue here https://github.com/webpack/webpack-cli/issues/new?template=Bug_report.md 
				`,
```

Reply:
Also use `here` instead of  `to` as in above suggestion

Reply:
Initially, I tried with template literal but it messes up the formatting. One can't just treat content inside template literal as it is and add a line break. It adds unwanted space, and if we remove that space then its alignment looks ugly with the rest of the code.

Reply:
Still I think it looks better that using `\n`.
I am Okay without template literals too if you like that way 🙂.
I have that unwanted spaces removed in suggestion if you like to consider.

Reply:
Yup, I noticed that. Initially tried that, too, but it just does not align with the rest of the code. 😉